Prepare release v2.1.0

mbanth opened the pull request.

xluciano committed:
Update README files for repo publication

xluciano committed:
Merge pull request #13 from xmos/feature/update_readme

Update README files for repo publication

danielpieczko committed:
Update to public licence

danielpieczko committed:
Update licence references in source files

danielpieczko committed:
Update changelog and version number

danielpieczko committed:
Merge pull request #15 from xmos/feature/licence_update

Update to public licence

danielpieczko committed:
Re-apply original copyright years

danielpieczko committed:
Merge pull request #16 from xmos/bugfix/original_copyright_years

Re-apply original copyright years

kieran-kohtz committed:
Delete LICENSE.rst

Licence not needed here, already in top level

kieran-kohtz committed:
Delete LICENSE.rst

Licence not needed in tests

kieran-kohtz committed:
Merge pull request #17 from xmos/feature/fix_licences

Feature/fix licences

xluciano reviewed (commented):
Looks good, but the copyright dates should be 2022, don't they?

xluciano reviewed (approved):
Probably the copyrights dates are ok, as the commits were created last year.

mbanth commented:
> Looks good, but the copyright dates should be 2022, don't they?

I had wondered about that also.  However, the file contents haven't change since last year, so I left the date ranges alone.
